modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -5,14 +5,16 @@
 
 		defaultConfig: {
 			mediaHitInterval: 30,
-			startTime:null
+			startTime:null,
```

Comment:
@MaksymKorniiov  how did this got in? 
Please fix this - it is not relevant to our ticket

Reply:
@eitanavgil  this is not my commit